applications/nrf_desktop/configuration/nrf52840_pca20041/app_ZDebug.conf

```diff
@@ -23,8 +23,8 @@ CONFIG_DESKTOP_MOTION_SENSOR_THREAD_STACK_SIZE=1024
 CONFIG_DESKTOP_MOTION_SENSOR_EMPTY_SAMPLES_COUNT=500
 CONFIG_DESKTOP_MOTION_SENSOR_CPI=1600
 CONFIG_DESKTOP_MOTION_SENSOR_SLEEP1_TIMEOUT_MS=500
-CONFIG_DESKTOP_MOTION_SENSOR_SLEEP2_TIMEOUT_MS=9220
-CONFIG_DESKTOP_MOTION_SENSOR_SLEEP3_TIMEOUT_MS=15000
+CONFIG_DESKTOP_MOTION_SENSOR_SLEEP2_TIMEOUT_MS=15000
+CONFIG_DESKTOP_MOTION_SENSOR_SLEEP3_TIMEOUT_MS=60000
```

Comment:
do not change this - instead "emulate" the disable of sleep3 on the second sensor by increasing the sleep3 timeout to maximum